client-participation/js/templates/vote-view.handlebars

```diff
@@ -330,9 +330,23 @@
     <div style="text-align: center; margin-bottom: 20px">
       <label class="weight-label">
         <span class="weight-radio"></span>
-        <input type="checkbox" name="checkbox" id="weight_high">
+        <input type="checkbox" name="checkbox" id="weight_high" aria-describedby="importantCheckboxDesc">
         {{s.importantCheckbox}}
+        <span id="importantCheckboxDesc" class="is-visually-hidden">
+          {{s.importantCheckboxDesc}}
+        </span>
       </label>
+      <i class="svgIcon" style="
+        display: inline-block;
+        position: relative;
+        top: 2px;
+        width: 17px;
+        fill: #8a8a8a;
+        "
+        data-text="{{s.importantCheckboxDesc}}"
+        onclick="alert(this.dataset.text)"
```

Comment:
Please replace the alert with a tooltip here. on-hover for desktop and on-tap for mobile.